### PR TITLE
Run3-hcx262 Modify the material budget analysis code for HCAL

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify the material budget analysis code for HCAL

#### PR validation:

Tested using the scripts in Validation/Geometry/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

File: Validation/Geometry/test/runP_HCAL_Debug_cfg.py [hunk 1]
```diff
@@ -67,6 +67,12 @@
         EtaHigh      = cms.untracked.double(3.0),
         EtaMinP      = cms.untracked.double(-5.5),
         EtaMaxP      = cms.untracked.double(5.5),
+        EtaMin0      = cms.untracked.double(2.650),
```
Comment (jfernan2) [inline Validation/Geometry/test/runP_HCAL_Debug_cfg.py:70]: May I suggest a more meaningul name for these variables? Something like etaHighMax, etaMidMax, etaLowMax,.... or at least some minimal order?

Comment (bsunanda) [reply Validation/Geometry/test/runP_HCAL_Debug_cfg.py]: Done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by jfernan2** [comment]
+1

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**